libobs/obs-nix.c

```diff
@@ -54,16 +54,15 @@ const char *get_module_extension(void)
 #define FLATPAK_PLUGIN_PATH "/app/plugins"
 
 static const char *module_bin[] = {
-	"../../obs-plugins/" BIT_STRING,
 	OBS_INSTALL_PREFIX "/" OBS_PLUGIN_DESTINATION,
+	"../../obs-plugins/" BIT_STRING,
 	FLATPAK_PLUGIN_PATH "/" OBS_PLUGIN_DESTINATION,
 };
 
 static const char *module_data[] = {
-	OBS_DATA_PATH "/obs-plugins/%module%",
 	OBS_INSTALL_DATA_PATH "/obs-plugins/%module%",
-	FLATPAK_PLUGIN_PATH "/share/obs/obs-plugins/%module%",
-};
+	OBS_DATA_PATH "/obs-plugins/%module%",
+	FLATPAK_PLUGIN_PATH "/share/obs/obs-plugins/%module%"};
```

Comment:
This formatting change appears to be a typo. To be consistent with the static declaration above the curly bracket should be on its own line.

Reply:
I think github is broken, these should not be marked as "Outdated"...

Reply:
This file is not modified in this PR. It may have been modified as part of one of its dependent PRs that has since been merged.